features/environment.py

```diff
@@ -24,9 +24,20 @@
 
 
 def before_all(context):
-    config = get_config(context.config.vendor)
 
-    context.config = config
+    plan = []
+    for feature in context._runner.features:
```

Comment:
I was assuming we wanted to create the whole scaffold for response, rather than just the count, so we'd have a grid with cells pre allocated.